app/models/transaction.rb

```diff
@@ -29,6 +31,7 @@ def total
   belongs_to :category, inverse_of: :transactions
   belongs_to :bank_account, inverse_of: :transactions
   has_one :organization, through: :bank_account, inverse_of: :transactions
+  has_one :customer, foreign_key: :id, primary_key: :customer_id
```

Comment:
@Gataniel let's keep organization as is, but I think we should change it for customer.